test/System/Clock.cpp

```diff
@@ -16,12 +16,53 @@ TEST_CASE("sf::Clock class - [system]")
         CHECK(clock.getElapsedTime() > elapsed);
     }
 
+    SUBCASE("hasTimePassed()")
+    {
+        const sf::Clock clock;
+        CHECK(clock.hasTimePassed(sf::microseconds(0)));
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
+        CHECK(clock.hasTimePassed(sf::milliseconds(1)));
+    }
+
+    SUBCASE("start()")
+    {
+        sf::Clock clock;
+        clock.start();
+        CHECK(clock.isRunning());
```

Comment:
Should we check that `isRunning` is false after construction?

Reply:
That's a good point since clock by default is running.